xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -305,9 +340,20 @@ viewers.Comments = Class.create({
                 }
               }
               this.destroyEditor("[name='" + name + "']", name);
+              this.formToggled = false;
```

Comment:
The reload of the editor doesn't imply the reload of the page: the parent dom shouldn't be hidden anymore?